osbuild/buildroot.py

```diff
@@ -196,7 +196,7 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None, extr
 
         # Import directories from the caller-provided root.
         imports = ["usr"]
-        if self.mount_boot:
+        if True:
```

Comment:
This is a part of a temporary patch which allows osbuild to use my host system as the build root. Colin made a recent change to the ostree-rs-ext repo that won't be in osbuild for a bit. Please disregard this change as it will not be present in the actual PR to osbuild.

Reply:
yeah, this part won't make it upstream - ignore.